app/code/Magento/Sitemap/Api/Data/SitemapInterface.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Sitemap\Api\Data;
+
+interface SitemapInterface extends SitemapExtensionInterface
+{
+
+    /**#@+
+     * Constants for keys of data array. Identical to the name of the getter in snake case
+     */
+    const SITEMAP_TIME      = 'sitemap_time';
+    const SITEMAP_TYPE      = 'sitemap_type';
+    const SITEMAP_FILENAME  = 'sitemap_filename';
+    const SITEMAP_PATH      = 'sitemap_path';
+    const STORE_ID          = 'store_id';
+    /**#@-*/
+
+    /**
+     * Get sitemap.xml URL according to all config options
+     *
+     * @param string $sitemapPath
+     * @param string $sitemapFileName
+     * @return string
+     */
+    public function getSitemapUrl($sitemapPath, $sitemapFileName): ?string;
```

Comment:
Remove `Sitemap` word from the all setters/getter

Reply:
Setter is missing for url

Reply:
getSitemapUrl was not plain dto logic so it has been removed from the interface.